From 1d86afb04c942b2ef17a07cf5895b7ff20d5385b Mon Sep 17 00:00:00 2001
From: Grigori Goronzy <greg@kinoho.net>
Date: Wed, 9 Feb 2022 22:20:17 +0100
Subject: [PATCH] dtoverlays: fix overrides in vc4-kms-dpi-generic

Apparently broken by commit 3c033975. Introduce new names to allow for
proper addressing of overrides.

Signed-off-by: Grigori Goronzy <greg@kinoho.net>
---
 .../overlays/vc4-kms-dpi-generic-overlay.dts   | 18 +++++++++---------
 1 file changed, 9 insertions(+), 9 deletions(-)

--- a/arch/arm/boot/dts/overlays/vc4-kms-dpi-generic-overlay.dts
+++ b/arch/arm/boot/dts/overlays/vc4-kms-dpi-generic-overlay.dts
@@ -12,7 +12,7 @@
 
 	fragment@0 {
 		target = <&panel>;
-		__overlay__  {
+		panel_generic: __overlay__  {
 			compatible = "panel-dpi";
 
 			width-mm = <154>;
@@ -40,7 +40,7 @@
 
 	fragment@1 {
 		target = <&dpi>;
-		__overlay__  {
+		dpi_node_generic: __overlay__  {
 			pinctrl-0 = <&dpi_18bit_gpio0>;
 		};
 	};
@@ -63,12 +63,12 @@
 		width-mm = <&panel>, "width-mm:0";
 		height-mm = <&panel>, "height-mm:0";
 
-		rgb565 = <&panel>, "bus-format:0=0x1017",
-			<&dpi_node>, "pinctrl-0:0=",<&dpi_16bit_gpio0>;
-		rgb666-padhi = <&panel>, "bus-format:0=0x1015",
-			<&dpi_node>, "pinctrl-0:0=",<&dpi_18bit_cpadhi_gpio0>;
-		rgb888 = <&panel>, "bus-format:0=0x100a",
-			<&dpi_node>, "pinctrl-0:0=",<&dpi_gpio0>;
-		bus-format = <&panel>, "bus-format:0";
+		rgb565 = <&panel_generic>, "bus-format:0=0x1017",
+			<&dpi_node_generic>, "pinctrl-0:0=",<&dpi_16bit_gpio0>;
+		rgb666-padhi = <&panel_generic>, "bus-format:0=0x1015",
+			<&dpi_node_generic>, "pinctrl-0:0=",<&dpi_18bit_cpadhi_gpio0>;
+		rgb888 = <&panel_generic>, "bus-format:0=0x100a",
+			<&dpi_node_generic>, "pinctrl-0:0=",<&dpi_gpio0>;
+		bus-format = <&panel_generic>, "bus-format:0";
 	};
 };
